pkg/service/generateConfig/generateConfig.go

```diff
@@ -108,5 +109,12 @@ func (g *generatorConfig) GenerateConfig(filePath string) {
 		g.logger.Fatal("Failed to write config file", zap.Error(err))
 	}
 
+
+  cmd := exec.Command("sudo", "chmod", "-R", "777", filePath)
```

Comment:
`err = os.WriteFile(filePath, results, os.ModePerm)` here os.ModePerm should give the read and write permissions. Firstly Check why it is not working properly or if there is any other permission mode which enables add it.

Reply:
is this clear ?

Reply:
yep